scripts/loop_client_tests.sh

```diff
@@ -14,8 +14,8 @@ cleanup() {
 
 cd safe_network/sn_client
 
-# run 10 times, increment counter once per iter
-for i in {0..10..1}
+# run 50 times, increment counter once per iter
+for i in {0..50..1}
```

Comment:
Oh wow, that's a nice boost in checks.
Will this run until success or actually soak it?

Reply:
so this just runs 50 tests regardless of outcome. We could set it much higher, but atm this has been enough to spot a mem leak